portal/component-tests/contract/components/contract-overview-table.component-test.js

```diff
@@ -85,9 +88,11 @@ describe(component, () => {
       .toRead('MIA Submission date'));
 
     it('displays deal.details.manualInclusionApplicationSubmissionDate', () => wrapper.expectText('[data-cy="submissionDate"]')
-      .toRead(moment(deal.details.dealWithManualInclusionApplicationSubmissionDate)
-        .tz('Europe/London')
-        .format('DD/MM/YYYY')));
+      .toRead(formatInTimeZone(
+        new Date(dealWithManualInclusionApplicationSubmissionDate.details.manualInclusionApplicationSubmissionDate),
```

Comment:
This previously worked because it was calling `deal(undefined)`. The mocked data also uses the current date so the day, month, year would all be the same